packages/shared.net/src/FetchAsJsonFn.ts

```diff
@@ -18,6 +18,7 @@
  * Fetches the given URL, converting the body to JSON, setting the Content-Type header to application/json,
  * and calling .json() on the response.
  */
+/** @deprecated */
 export type FetchAsJsonFn = (
```

Comment:
When did we deprecate this?

Reply:
its not used but im playing nice with semver